SPECS/cpufrequtils/cpufrequtils.spec

```diff
@@ -0,0 +1,206 @@
+Summary:        CPU Frequency changing related utilities
+Name:           cpufrequtils
+Version:        008
+Release:        5%{?dist}
+Group:          System Environment/Base
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://mirrors.edge.kernel.org/pub/linux/utils/kernel/cpufreq/
+Source:         https://mirrors.edge.kernel.org/pub/linux/utils/kernel/cpufreq/%{name}-%{version}.tar.gz
+Patch0:         disable-gsic.patch
+Patch1:         cpufrequtils-multilib.patch
+Patch2:         cpufrequtils-008-aperf-32bit.patch
+Patch50:        cpufrequtils-008-no-aperf-on-ppc.patch
+BuildRequires:  sysfsutils
+BuildRequires:  gettext
+# pulls in automake and autoconf
+BuildRequires:  libtool
+ExclusiveArch:  %{ix86} x86_64 ppc ppc64
+Provides:       cpufreq-utils = 1:%{version}-%{release}
+Obsoletes:      cpufreq-utils < 1:%{version}-%{release}
+ 
+%description
+cpufrequtils contains several utilities that can be used to control
+the cpufreq interface provided by the kernel on hardware that
+supports CPU frequency scaling.
+ 
+%package devel
+Summary:        CPU frequency changing utilities development files
+Group:          Development/Libraries
+License:        GPLv2+
+ 
+%description devel
+The cpufrequtils development files.
+ 
+%prep
+%setup -q
+%patch 0 -p1
+%patch 1 -p1
+%patch 2 -p1 -b .aperf-32bit
+%ifarch ppc ppc64
+%patch 50 -p1 -b .no-aperf-on-ppc
+%endif
+ 
+%build
+make CFLAGS="$RPM_OPT_FLAGS" DEBUG=true
+%ifarch %{ix86}
+cd debug/i386
+make CFLAGS="$RPM_OPT_FLAGS"
+%endif
+%ifarch x86_64
+cd debug/x86_64
+make CFLAGS="$RPM_OPT_FLAGS"
+%endif
+cd ..
+ 
+%install
+make DESTDIR=%{buildroot} mandir=%{_mandir} bindir=%{_bindir} includedir=%{_includedir} libdir=%{_libdir} install
+ 
+# Remove libtool lib and static lib
+rm -f %{buildroot}%{_libdir}/*.{a,la}
+ 
+%find_lang cpufrequtils
+ 
+%ifarch %{ix86}
+cd debug/i386
+install centrino-decode %{buildroot}%{_bindir}/centrino-decode
+install dump_psb %{buildroot}%{_bindir}/dump_psb
+install powernow-k8-decode %{buildroot}%{_bindir}/powernow-k8-decode
+cd ../..
+%endif
+%ifarch x86_64
+cd debug/x86_64
+install powernow-k8-decode %{buildroot}%{_bindir}/powernow-k8-decode
+cd ../..
+%endif
+ 
+chmod -R a-s %{buildroot}
+ 
+%post -p /sbin/ldconfig
+ 
+%postun -p /sbin/ldconfig
+ 
+%files -f cpufrequtils.lang
+%defattr(-,root,root,0755)
+%doc COPYING
+%{_libdir}/libcpufreq.so*
+%{_bindir}
+%{_mandir}/*/*
+ 
+%files devel
+%defattr(-,root,root,0755)
+%doc COPYING
+%{_includedir}/cpufreq.h
+ 
+%changelog
+
+* Mon Jul 22 2024 Aditya Dubey <adityadubey@microsoft.com> - 008-5
+- Onboarding package to azurelinux
```

Comment:
This needs to be phrased as `Initial Azure Linux import from Fedora (replace_with_fedora_version)` so that the tool can pick up the source of the spec file

Reply:
And you need to verify if the license is correct and add another `- License verified` line in the changelog below this one